proxy/src/transport/tls/config.rs

```diff
@@ -45,7 +45,10 @@ pub struct CommonSettings {
 
     /// The identity we use to identify the service being proxied (as opposed
```

Comment:
Is this comment still correct (i.e. should it now say "...to identify the _pod_ being proxied...")?

Reply:
I changed it to "pod". Thanks! (It's unfortunate we don't have a great term to use that's not Kubernetes-specific but which allows us to immediately tell whether we're talking about a Kuernetes pod vs. a Kubernetes service.)